docs/developer_guide.md

```diff
@@ -296,6 +296,27 @@ make ovms_builder_image
 make run_unit_tests
 ```
 
+To run unit tests verifying integration with Intel GPUs, add `RUN_GPU_TESTS=ON` parameter:
+```
+make run_unit_tests RUN_GPU_TESTS=ON
```

Comment:
what param should be added to `bazel test` command? It will be more useful than running all tests

Reply:
more useful for who? for validation team we need to ensure that make run_unit_tests includes gpu device and installs va_api dependencies inside builder image. impossible via bazel env

i will add to documentation how to run gpu unit tests when builder image is already set up, it will be `--test_env RUN_GPU_TESTS=1`